aten/src/ATen/native/cpu/SortingKernel.cpp

```diff
@@ -19,6 +19,9 @@
 #ifdef USE_FBGEMM
 #include <fbgemm/Utils.h>
 #endif
+#if __has_include(<parallel/algorithm>) && defined(_OPENMP)
+#include <parallel/algorithm>
```

Comment:
Is there a C++ standard that says this header will be available in C++20? Or C++23? If not, please don't rely on this header